builtin_metadata.json

```diff
@@ -8726,6 +8753,26 @@
     },
     "wasm": true
   },
+  "jsonschema.verify": {
```

Comment:
Should this perhaps be `json.verify_schema`?

Reply:
Yes `json.match_schema` and `json.verify_schema` would be better.